src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -26,13 +26,35 @@ public class CrossTenantQueryService {
 
   private static final String COMPOSITE_INSTANCES_ID = "6b08439b-4f8e-4468-8046-ea620f5cfb74";
   private static final String SIMPLE_INSTANCES_ID = "8fc4a9d2-7ccf-4233-afb8-796911839862";
+  private static final String SIMPLE_INSTANCE_STATUS_ID = "9c239bfd-198f-4013-bbc4-4551c0cbdeaa";
+  private static final String SIMPLE_INSTANCE_TYPE_ID = "af44e2e0-12e0-4eec-b80d-49feb33a866c";
+  private static final List<String> INSTANCE_RELATED_ENTITIES = List.of(SIMPLE_INSTANCES_ID, COMPOSITE_INSTANCES_ID, SIMPLE_INSTANCE_STATUS_ID, SIMPLE_INSTANCE_TYPE_ID);
```

Comment:
Contains all entity types related to composite_instances. We use this list when retrieving column values, so that we are able to provide cross-tenant values for all composite instance fields that use a dropdown (even though the simple entity type itself is not cross-tenant).